Source/WebKit/NetworkProcess/NetworkProcessCreationParameters.h

```diff
@@ -42,6 +43,18 @@
 
 namespace WebKit {
 
+struct NetworkProcessCreationParametersMarkableDoubleTraits {
```

Comment:
Can we define this in Markable.h, next to the Traits for `float` and get rid of `WebAnimationsMarkableDoubleTraits` in the process?

Reply:
I've moved the traits to Markable.h. Seems like we can't eliminate the second traits parameter without more involved changes for now. But as discussed, I've edited the test to allow this to work.